File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ObtenerEstadisticasOut.ts
import { ItemValor } from '../../Components/Estadistica/class/itemValor';
import { ParametroBaseOut } from './ParametroBaseOut';
export class ObtenerEstadisticasOut extends ParametroBaseOut{

    private datos: ItemValor[];

    constructor()
    {
        super();
        this.datos = [];
    }

    public get Datos(): ItemValor[] {
        return this.datos;
    }
    public set Datos(value: ItemValor[]) {
        this.datos = value;
    }

}

File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ObtenerEstadisticasOut.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ObtenerEstadisticasOut : ParametroBaseOut
    {
        public List<ItemValor> Datos { get; set; }
    }
}